docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -215,7 +215,7 @@ Using this annotation you can add additional configuration to the NGINX location
 
 ```yaml
 nginx.ingress.kubernetes.io/configuration-snippet: |
-  more_set_headers "Request-Id: $req_id";
+  more_set_input_headers "X-Request-ID: $request_id";
```

Comment:
We have a mapping at https://github.com/kubernetes/ingress-nginx/blob/master/rootfs/etc/nginx/template/nginx.tmpl#L278 that takes care of setting `$req_id`. It also makes sure if `X-Request-ID` exists then we use that otherwise use `$request_id` to generate new one. Therefore I think this example should stick to `$req_id`.

Reply:
Also this example is a bit redundant because we now always set `X-Request-Id`: https://github.com/kubernetes/ingress-nginx/pull/2446/files. One does not need custom snippet for that anymore.

Maybe come up with a different, more frequently requested example?

Reply:
Ok thank you for the review i was not aware of these recent changes. The sample did not work with my helm chart (version quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.13.0) and I thought that would be an improvent. 
I will submit other changes when stumble over a similar issue and the cooperate CLA is ready.

Reply:
Thank you!